pkg/kubelet/cm/container_manager_linux.go

```diff
@@ -240,6 +290,15 @@ func (cm *containerManagerImpl) setupNode() error {
 		return err
 	}
 
+	// Setup top level qos containers only if CgroupsPerQOS flag is specified as true
+	if cm.NodeConfig.CgroupsPerQOS {
+		qosContainersInfo, err := InitQOS(cm.NodeConfig.CgroupRoot, cm.subsystems)
```

Comment:
for your own future sanity, i think you will want to ensure that these are never deleted by a rogue init system.  its fine to handle that in a future todo if @vishh agrees.